frontend/src/components/associations/marketplace/counter/CounterOrderMaker.tsx

```diff
@@ -103,38 +103,61 @@ export const CounterOrderMaker = ({
             });
         }
 
-        queryCache.refetchQueries("marketplace.transactions.list");
+        await queryCache.refetchQueries("marketplace.transactions.list");
     };
 
+    const basketIsEmpty = Object.keys(basket).length === 0;
+
     return (
         <>
-            <Card>
-                <Card.Header>
-                    Solde :{" "}
-                    {balanceStatus === "loading" ? (
-                        <em>Chargement</em>
-                    ) : balanceStatus === "error" ? (
-                        <em>Erreur {balanceError} : balance</em>
-                    ) : (
-                        `${(balance as { balance: number }).balance}€`
-                    )}
-                </Card.Header>
-            </Card>
-
-            <RefundForm customer={customer} marketplaceId={marketplaceId} />
-
-            <ProductSearch
-                basket={basket}
-                addToBasket={addToBasket}
+            <RefundForm
+                customer={customer}
                 marketplaceId={marketplaceId}
+                className={"float-right mt-2"}
             />
 
-            {/* Show order summary */}
-            <OrderSummary
-                basket={basket}
-                removeFromBasket={removeFromBasket}
-                makeOrder={makeOrder}
-            />
+            <div className="d-flex justify-content-center mt-4">
+                <Col md={"6"}>
+                    <Card>
+                        <Card.Body className="text-center">
+                            <h5>{customer.id}</h5>
+                            <div className="display-2 font-weight-bold">
+                                {balanceStatus === "loading" ? (
+                                    <em>Chargement</em>
+                                ) : balanceStatus === "error" ? (
+                                    <em>Erreur {balanceError} : balance</em>
+                                ) : (
+                                    `${
+                                        (balance as { balance: number }).balance
+                                    }€`
+                                )}
+                            </div>
+                        </Card.Body>
+                    </Card>
+                </Col>
+            </div>
+
+            <Row>
+                <Col md={basketIsEmpty ? 12 : 6}>
+                    <ProductSearch
+                        basket={basket}
+                        addToBasket={addToBasket}
+                        marketplaceId={marketplaceId}
+                        compressed={Object.keys(basket).length > 0}
+                    />
+                </Col>
+
+                {/* Show order summary */}
+                {!basketIsEmpty && (
```

Comment:
I think the UI should not change so much depending on `basketIsEmpty`.

Example: imagine you want to select the item 11 but select a wrong product (here item 13).

![image](https://user-images.githubusercontent.com/11032610/83331650-98783e80-a297-11ea-88f8-86b9d33c6a04.png)

This is the screen when you are about to click on the red button to remove the item 13 from your order. And you already know the position of item 11, so you know where to click just after removing item 13.

But… the UI changes. And you might, in the end, select _another_ wrong item.

![image](https://user-images.githubusercontent.com/11032610/83331696-ed1bb980-a297-11ea-8287-173cf126e711.png)

Reply:
Also, please add the pointer cursor on the cards